src/crypto/randomx/randomx.cpp

```diff
@@ -70,6 +70,36 @@ RandomX_ConfigurationWownero::RandomX_ConfigurationWownero()
 	fillAes4Rx4_Key[7] = fillAes4Rx4_Key[3];
 }
 
+RandomX_ConfigurationARQ::RandomX_ConfigurationARQ()
+{
+	ArgonSalt = "RandomARQ\x01";
+	ProgramIterations = 1024;
+	ProgramSize = 256;
+	ProgramCount = 4;
+	ArgonIterations = 1;
+	ArgonLanes = 1;
+	ScratchpadL2_Size = 131072;
+	ScratchpadL3_Size = 262144;
+
```

Comment:
You only need to assign values that are different from the default (Monero) config. As far as I can see, it's RANDOMX_ARGON_SALT, RANDOMX_PROGRAM_ITERATIONS, RANDOMX_PROGRAM_COUNT, RANDOMX_SCRATCHPAD_L3, RANDOMX_SCRATCHPAD_L2. Everything else is the same and doesn't need to be in this constructor.